codegen/src/test/resources/software/amazon/awssdk/codegen/poet/transform/streaminginputoperationrequestmodelmarshaller.java

```diff
@@ -0,0 +1,36 @@
+package software.amazon.awssdk.services.jsonprotocoltests.transform;
+
+import javax.annotation.Generated;
+import software.amazon.awssdk.annotations.SdkInternalApi;
+import software.amazon.awssdk.core.exception.SdkClientException;
+import software.amazon.awssdk.core.protocol.ProtocolMarshaller;
+import software.amazon.awssdk.services.jsonprotocoltests.model.StreamingInputOperationRequest;
+import software.amazon.awssdk.utils.Validate;
+
+/**
+ * {@link StreamingInputOperationRequest} Marshaller
+ */
+@Generated("software.amazon.awssdk:codegen")
+@SdkInternalApi
+public class StreamingInputOperationRequestModelMarshaller {
+    private static final StreamingInputOperationRequestModelMarshaller INSTANCE = new StreamingInputOperationRequestModelMarshaller();
+
+    private StreamingInputOperationRequestModelMarshaller() {
+    }
+
+    public static StreamingInputOperationRequestModelMarshaller getInstance() {
+        return INSTANCE;
+    }
+
+    /**
+     * Marshall the given parameter object
+     */
+    public void marshall(StreamingInputOperationRequest streamingInputOperationRequest, ProtocolMarshaller protocolMarshaller) {
+        Validate.paramNotNull(streamingInputOperationRequest, "streamingInputOperationRequest");
+        Validate.paramNotNull(protocolMarshaller, "protocolMarshaller");
+        try {
```

Comment:
This doesn't seem right, shouldn't it be marshalling the streaming member?

Reply:
don't we use [StreamingRequestMarshaller](https://github.com/aws/aws-sdk-java-v2/blob/master/core/src/main/java/software/amazon/awssdk/core/runtime/transform/StreamingRequestMarshaller.java) to marshall all streaming members in v2?